simd/src/Kokkos_SIMD.hpp

```diff
@@ -62,7 +62,8 @@
 #include <Kokkos_SIMD_AVX512.hpp>
 #endif
 
-#if defined(KOKKOS_ARCH_ARM_NEON)
+// FIXME: If and when NVCC supports NEON intrinsics reenable
+#if defined(KOKKOS_ARCH_ARM_NEON) && !defined(__NVCC__)
```

Comment:
Do we still need this? If you don't compile Kokkos with `nvcc` then you can't reasonably use `nvcc` in downstream code anyway.

Reply:
It might be helpful for makefile users. But I can remove it. It was just as is from #6823.

Reply:
We do not need the `_NVC_` check here if we use the check in CMake, I added.

Reply:
I would prefer to use the CMake check to be conform with the check for AVX when using nvc.

Reply:
You can merge/rebase this [commit](https://github.com/kokkos/kokkos/pull/6823/files/be4c315b67063e7c820788d78971883f55f8e30c..089f2a808d87fe07abd67d9d3315f8cb24e54b30#diff-a3f5ed9cb1b4c0bd845f07380bd1bfb04818ffd9d12ef1c944e7797dea4d7f06) for the CMake checl. This worked at LANL and the system is Switzerland.

Reply:
It is already there!
I already took everything you've done from the other branch ;).